src/extension/options-page/DashboardDialogs/Feedback.tsx

```diff
@@ -45,8 +47,10 @@ export function DashboardFeedbackDialog(props: WrappedDialogProps) {
                 footer={
                     <DebounceButton
                         variant="contained"
-                        disabled={Boolean(!name || !email || !message)}
-                        onClick={async () => {}}>
+                        disabled={Boolean(!name || !message)}
+                        onClick={async () => {
+                            window.open(`mailto:info@dimension.im?body=${message}&subject=${name}`)
```

Comment:
use URL builder this string.

Reply:
`new URL()` convert string to object, not the other way around.

Reply:
```
const url = new URL("mailto:info@dimension.im")
url.searchParams.set("subject", "subject")
url.searchParams.set("body", "💊")
url.toString()
// returns "mailto:info@dimension.im?subject=subject&body=%F0%9F%92%8A"
```

join strings, you have not url encode